Sources/Basics/DispatchTimeInterval+Extensions.swift

```diff
@@ -0,0 +1,59 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2020 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+ */
+
+import Dispatch
+import struct Foundation.TimeInterval
+
+extension DispatchTimeInterval {
```

Comment:
This seems generally useful.  Should it perhaps be marked `public` so it can be used outside the module, or not yet?